website/docs/ssh/README.md

```diff
@@ -88,7 +122,77 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs>
-<TabItem value="terminal" label="Terminal" default>
+<TabItem value="linux" label="Linux" default>
+
+Open a terminal window and type the following command:
+
+```bash
+gedit ~/.ssh/config # This will open your .ssh/config
+# Here we're using gedit as our text manager, feel free to choose another one (e.g., vi, nano)
+```
+
+Hit **Enter** and then add the following lines to your file:
+
+```bash
+Host <robot-name>
+  HostName <ip-address>
+  User farm-ng-user-<username>
+  ```
+
+Replace `<robot-name>` with your robot name (e.g., element-vegetable) and
+`<ip-address>` the robot's local IP address.
+
+Replace `<username>` with your username (e.g., john-doe).
+Make sure to save the file before you close it!
+
+:::tip TIP
+You can download [**Fing**](https://www.fing.com/products/fing-app) to scan
+all devices connected to your network and figure out your robot's IP address.
+:::
+</TabItem>
+
+<TabItem value="windows" label="Windows" default>
+
+a. Open the command prompt (Windows Key + R -> cmd -> Enter)
+
+b. On the terminal type:
+
+```bash
+cd .ssh
+start .
+```
+
+c. Hit **Enter**
+
+d. This will open the folder where you need to save your configurations.
+
+e. Create a new file and name it config.
+
+**Hint**: Create a copy of your ssh_rsa.pub, and rename it to config;
+but make sure that it doesn't have an extension (e.g., ".pub").
+
```

Comment:
Not a big fan of this hint
```suggestion
```

Reply:
Proposed alternative in [34928e6](https://github.com/farm-ng/amiga-dev-kit/pull/216/commits/34928e6fd63611cac9da8019e184409b50f789cf)